### PR TITLE
Removed unused and redundant sprocket assets [Sprocket Migration 1/?]

### PR DESCRIPTION
**Pull request by FinnIckler**
This PR removes
- The Tutorial Images for the Workbook assistant
- Google Maps code (including locationpicker.jquery)
- All uses of Glyphicon (but I can't find where it is actually referenced?)
- A css file called pageMenu2.css which gets used nowwhere

### CODE REVIEW

**Comment by FinnIckler** [comment]
We are not using it in Sprockets though, if you look through the assets files in the repo they are empty

**Comment by gregorbg** [comment]
Adding it here requires it from the gem, see instructions under "Installation" at https://github.com/starburstgem/starburst

**Comment by FinnIckler** [comment]
I added it back in, very weird that this is needed considering everything that is in that asset folder in the gem is https://github.com/starburstgem/starburst/blob/master/app/assets/javascripts/starburst/starburst.js

File: WcaOnRails/app/assets/stylesheets/collapsible-panels.scss [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .collapse-indicator {
-  @extend .glyphicon;
-  @extend .glyphicon-chevron-down;
+  @extend .icon;
+  content: "\f078"; // Semantic chevron down
 
   .collapsed & {
-    @extend .glyphicon-chevron-right;
```
Comment (gregorbg) [inline WcaOnRails/app/assets/stylesheets/collapsible-panels.scss:6]: It used to be `chevron-right`, but you're picking "chevron up" as a replacement.
I think it should be `f054` for "chevron right": https://github.com/fomantic/Fomantic-UI-SASS/blob/master/app/assets/stylesheets/semantic-ui/elements/_icon.scss#L1299

File: WcaOnRails/app/helpers/application_helper.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@ def link_to_competition_schedule_tab(comp)
 
   def anchorable(pretty_text, id = nil)
     id ||= pretty_text.parameterize
-    "<span id='#{id}' class='anchorable'><a href='##{id}'><span class='glyphicon glyphicon-link'></span></a> #{pretty_text}</span>".html_safe
+    "<span id='#{id}' class='anchorable'><a href='##{id}'><span class='linkify icon'></span></a> #{pretty_text}</span>".html_safe
```
Comment (gregorbg) [inline WcaOnRails/app/helpers/application_helper.rb:48]: Rather `icon linkify` for clarity?

File: WcaOnRails/app/views/admin/avatars/index.html.erb [hunk 3]
```diff
@@ -86,11 +86,11 @@
 
                     <!-- Controls -->
                     <a class="left carousel-control" href="#carousel-<%= user.id %>-old-avatars" role="button" data-slide="prev">
-                      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+                      <span class="chevron left icon" aria-hidden="true"></span>
```
Comment (gregorbg) [inline WcaOnRails/app/views/admin/avatars/index.html.erb:89]: `icon` first

File: WcaOnRails/app/views/admin/avatars/index.html.erb [hunk 4]
```diff
@@ -86,11 +86,11 @@
...
                       <span class="sr-only">Previous</span>
                     </a>
                     <a class="right carousel-control" href="#carousel-<%= user.id %>-old-avatars" role="button" data-slide="next">
-                      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+                      <span class="chevron right icon" aria-hidden="true"></span>
```
Comment (gregorbg) [inline WcaOnRails/app/views/admin/avatars/index.html.erb:93]: `icon` first

File: WcaOnRails/app/views/polls/_polls_list.html.erb [hunk 5]
```diff
@@ -4,7 +4,7 @@
       <td><%=md poll.question %></td>
       <td>
         <% if poll.user_already_voted?(current_user) %>
-          <span class="glyphicon glyphicon-ok-sign" aria-hidden="true" data-toggle="tooltip" data-placement="top" data-container="body" title="You have already voted in this poll"></span>
+          <span class="check circle icon" aria-hidden="true" data-toggle="tooltip" data-placement="top" data-container="body" title="You have already voted in this poll"></span>
```
Comment (gregorbg) [inline WcaOnRails/app/views/polls/_polls_list.html.erb:7]: `icon` first

File: WcaOnRails/app/views/posts/_post.html.erb [hunk 6]
```diff
@@ -9,11 +9,11 @@
       <% if current_user&.can_create_posts? %>
         <span class="pull-right">
           <%= link_to edit_post_path(post.slug) do %>
-            <span class="glyphicon glyphicon-pencil"></span>
+            <span class="pencil alternate icon"></span>
```
Comment (gregorbg) [inline WcaOnRails/app/views/posts/_post.html.erb:12]: `icon` first

File: WcaOnRails/app/views/posts/_post.html.erb [hunk 7]
```diff
@@ -9,11 +9,11 @@
...
           <% end %>
           <% if post.persisted? %>
             <%= link_to post_path(post.slug), method: "delete", data: { confirm: I18n.t('posts.confirm_delete_post') } do %>
-              <span class="glyphicon glyphicon-trash"></span>
+              <span class="trash icon"></span>
```
Comment (gregorbg) [inline WcaOnRails/app/views/posts/_post.html.erb:16]: `icon` first

File: WcaOnRails/app/views/posts/_post_form.html.erb [hunk 8]
```diff
@@ -21,7 +21,7 @@
 
   <% if @post.persisted? %>
     <%= link_to post_path(@post.slug), method: "delete", data: { confirm: I18n.t('posts.confirm_delete_post') }, class: "btn btn-danger" do %>
-      <span class="glyphicon glyphicon-trash"></span> Delete post
+      <span class="trash icon"></span> Delete post
```
Comment (gregorbg) [inline WcaOnRails/app/views/posts/_post_form.html.erb:24]: `icon` first

File: WcaOnRails/spec/helpers/markdown_helper_spec.rb [hunk 9]
```diff
@@ -16,7 +16,7 @@
 <a href="#foo-bar">Foo Bar</a>
 </li>
 </ul>
-<h1><span id=\'foo-bar\' class=\'anchorable\'><a href=\'#foo-bar\'><span class=\'glyphicon glyphicon-link\'></span></a> Foo Bar</span></h1>
+<h1><span id=\'foo-bar\' class=\'anchorable\'><a href=\'#foo-bar\'><span class=\'linkify icon\'></span></a> Foo Bar</span></h1>
```
Comment (gregorbg) [inline WcaOnRails/spec/helpers/markdown_helper_spec.rb:19]: `icon` first

**Review by gregorbg — CHANGES_REQUESTED** [review]
Nitpicks about the CSS styles

File: WcaOnRails/app/assets/stylesheets/collapsible-panels.scss (hunk 1 above)
Comment (FinnIckler) [reply WcaOnRails/app/assets/stylesheets/collapsible-panels.scss:6]: not sure why you linked `i.icon.cloud.meatball` but I changed it ;) to [chevron right](https://github.com/fomantic/Fomantic-UI-SASS/blob/87a7b23c5d6a598043d13f32a2e9875d6b8f80f2/app/assets/stylesheets/semantic-ui/elements/_icon.scss#L1252)

**Review by gregorbg — APPROVED** [review]

**Pull request merged**